schemas/hotel-details/index.ts

```diff
@@ -99,6 +99,36 @@ export const details: FeatureSchemaDefinition = {
         title: "[Details] Room Specification",
         value: "details.card.room-specification",
       },
+      {
+        title:
+          "[Details] Right Media Aspect Ratio[2:4] with Top Left Layover content",
+        value: "details.card.right-media-ar-2:4-with-top-left-layover-content",
```

Comment:
what is ar ?
Provide a valid nomenclature for value

Reply:
ill resolve it